guides/common/modules/proc_configuring-repositories-foreman-deb.adoc

```diff
@@ -0,0 +1,24 @@
+.Procedure
+Select the operating system and version you are installing on:
+
+* xref:#repositories-debian-11[Debian 11 (Bullseye)]
+* xref:#repositories-ubuntu-2004[Ubuntu 20.04 (Focal)]
+* xref:#repositories-ubuntu-2204[Ubuntu 22.04 (Jammy)]
```

Comment:
This should still give no diff, right? If so, it's another way to make maintenance easier.

```suggestion
* xref:#repositories-debian-11[]
* xref:#repositories-ubuntu-2004[]
* xref:#repositories-ubuntu-2204[]
```